benchmarks/strings/iteration_bench.nit

```diff
@@ -62,44 +45,12 @@ do
 	end
 end
 
-fun bench_ropestr_iter(nb_cct: Int, loops: Int, strlen: Int)
-do
-	var a = "a" * strlen
-	var x: String = new Concat(a, a)
-	for i in [2 .. nb_cct[ do x = new Concat(x, a)
-	var cnt = 0
-	var c: Char
-	while cnt != loops do
-		for i in x do
-			c = i
-		end
-		cnt += 1
-	end
-end
-
-fun bench_ropestr_index(nb_cct: Int, loops: Int, strlen: Int)
-do
-	var a = "a" * strlen
-	var x: String = new Concat(a, a)
-	for i in [2 .. nb_cct[ do x = new Concat(x, a)
-	var cnt = 0
-	var c: Char
-	var pos = 0
-	while cnt != loops do
-		pos = 0
-		while pos < x.length do
-			c = x[pos]
-			pos += 1
-		end
-		cnt += 1
-	end
-end
-
-fun bench_flatbuf_iter(nb_cct: Int, loops: Int, strlen: Int)
+private fun bench_buffer_iter(nb_cct: Int, loops: Int, strlen: Int)
 do
 	var a = "a" * strlen
 	a = a * nb_cct
```

Comment:
the initial creation of stings should be factorized and simplified (and possibly being another dimension of the tests).